tests/cli/test_config_validation_hints.py

```
@@ -0,0 +1,99 @@
+    Bypass the global setup gate too — this test is about validation
+    error formatting, not the gate.
+    """
+    monkeypatch.setattr("config.manager.DEFAULT_CONFIG_DIR", tmp_path)
+    monkeypatch.setattr("cli.organize._check_setup_completed", lambda: True)
+    return tmp_path
+
+
+@pytest.mark.integration
+@pytest.mark.ci
+def test_config_edit_invalid_device_includes_valid_values(
+    isolated_config: Path,
```

Ruff (0.15.12)

[warning] Unused function argument: `isolated_config`

(ARG001)